Re-enable WebTransport tests on iOS

achristensen07 opened the pull request:
#### 76f07698b7ae6b7d900bc00041dcf6888058fea7
<pre>
Re-enable WebTransport tests on iOS
<a href="https://bugs.webkit.org/show_bug.cgi?id=297865">https://bugs.webkit.org/show_bug.cgi?id=297865</a>
<a href="https://rdar.apple.com/148050136">rdar://148050136</a>

Unreviewed.

They were definitely crashing a few months ago, and they definitely aren&apos;t any more.

* Tools/TestWebKitAPI/Tests/WebKitCocoa/WebTransport.mm:
(TestWebKitAPI::TEST(WebTransport, ClientBidirectional)):
(TestWebKitAPI::TEST(WebTransport, Datagram)):
(TestWebKitAPI::TEST(WebTransport, Unidirectional)):
(TestWebKitAPI::TEST(WebTransport, NetworkProcessCrash)):
(TestWebKitAPI::TEST(WebTransport, Worker)):
(TestWebKitAPI::TEST(WebTransport, DISABLED_ClientBidirectional)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_Datagram)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_Unidirectional)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_NetworkProcessCrash)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_Worker)): Deleted.

Canonical link: <a href="https://commits.webkit.org/299128@main">https://commits.webkit.org/299128@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d4c6d4fd2026e6424e638a123bb80d88a80a7b05

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/117957 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/37635 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/28271 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/124104 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/69992 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/e4507db0-9d81-46f5-a88a-6aca5f330760) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/119835 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/38328 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/46217 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/89513 "Passed tests") | [💥 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/59120 "An unexpected error occured. Recent messages:Printed configuration") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/ffb614b4-65aa-4703-bcf6-94d2435d8bed) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/120909 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/30525 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/105776 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/70006 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/c57c1a35-e4f8-468e-89bf-2d0239b48c48) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/29592 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/23891 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/67767 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/99947 "Passed tests") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/24070 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/127185 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/44860 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/33796 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/98179 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/45221 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/102000 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/97967 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/43369 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/21355 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/41294 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18805 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/44732 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/44192 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/47537 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/45881 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/d4c6d4fd2026e6424e638a123bb80d88a80a7b05. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49850

achristensen07 committed:
Re-enable WebTransport tests on iOS
https://bugs.webkit.org/show_bug.cgi?id=297865
rdar://148050136

Unreviewed.

They were definitely crashing a few months ago, and they definitely aren't any more.

* Tools/TestWebKitAPI/Tests/WebKitCocoa/WebTransport.mm:
(TestWebKitAPI::TEST(WebTransport, ClientBidirectional)):
(TestWebKitAPI::TEST(WebTransport, Datagram)):
(TestWebKitAPI::TEST(WebTransport, Unidirectional)):
(TestWebKitAPI::TEST(WebTransport, NetworkProcessCrash)):
(TestWebKitAPI::TEST(WebTransport, Worker)):
(TestWebKitAPI::TEST(WebTransport, DISABLED_ClientBidirectional)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_Datagram)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_Unidirectional)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_NetworkProcessCrash)): Deleted.
(TestWebKitAPI::TEST(WebTransport, DISABLED_Worker)): Deleted.

Canonical link: https://commits.webkit.org/299128@main
